30-queues_stacks.cs

```diff
@@ -0,0 +1,35 @@
+string? text = Console.ReadLine();
+
+PalidromeSoulation instancePalidromeSoulation = new PalidromeSoulation();
+
+foreach (char symbol in text)
+{
+    instancePalidromeSoulation.PushCharacter(symbol);
+    instancePalidromeSoulation.EnqueueCharacter(symbol);
+}
+
+bool isPalindrome = true;
+
+for (int i = 0; i < text.Length / 2; i++)
+{
+    if (instancePalidromeSoulation.PopCharacter() != instancePalidromeSoulation.DequeueCharacter())
+    {
+        isPalindrome = false;
+
+        break;
+    }
+}
+
+Console.WriteLine(isPalindrome ? $"The word, {text}, is a palindrome."
+    : $"The word, {text}, is not a palindrome.");
+
```

Comment:
What about  `Console.WriteLine($"The word, {text}, is {isPalindrome ? "" : " not" } a palindrome.");` ???